fs/fstestutil/record/record.go

```diff
@@ -1,11 +1,11 @@
-package record // import "bazil.org/fuse/fs/fstestutil/record"
+package record // import "github.com/suffuse/fuse/fs/fstestutil/record"
```

Comment:
It's something you say I should have done. Removing the comment is a manual change. The  commit has no manual changes.